optuna/terminator/callback.py

```diff
@@ -13,6 +13,55 @@
 
 @experimental_class("3.2.0")
 class TerminatorCallback:
+    """A callback that terminates the optimization using Terminator.
+
+    This class implements a callback which wraps :class:`~optuna.terminator.Terminator`
+    so that it can be used with the :func:`~optuna.study.Study.optimize` method.
+
+    Args:
+        terminator:
+            A terminator object which determines whether to terminate the optimization by
+            assessing the room for optimization and statistical error. Defaults to a
+            :class:`~optuna.terminator.Terminator` object with default
```

Comment:
Maybe we should explain here what the default `improvement_evluator` and `error_evaluator` does, because for most users this is the only interface to touch?

Reply:
I think we should not explain them here. It is enough to explain them in the document of `Terminator`.